CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.4.1 (2019-08-07)
+
+- Fixed the repeater item component to properly align selction checkboxes and context menus. [#55](https://github.com/blackbaud/skyux-lists/pull/55)
+- Fixed the repeater item component to not show the collapse chevron when then the item cannot collapse. [#56](https://github.com/blackbaud/skyux-lists/pull/56)
```

Comment:
Fixed the repeater item component to hide the collapse chevron when items cannot collapse.

Is that still accurate?

Reply:
Ya. This was for a bug where there were times where even when "expandMode" was "none" the chevron would show up.

Reply:
I worded my question poorly. I meant is my proposed edit accurate, and if yes, let's go with that please 😬

Reply:
Done